engine/src/main/java/org/pentaho/di/core/util/PluginPropertyHandler.java

```diff
@@ -237,25 +237,25 @@ protected void handle( final PluginProperty property ) throws KettleException {
    */
   public static class SaveToPreferences extends AbstractHandler {
 
-    private final Preferences node;
+    private final Preferences preferences;
 
     /**
      * Constructor.
      *
-     * @param node
-     *          node to set.
+     * @param preferences
+     *          preferences to set.
```

Comment:
Just for future reference- I personally find no value in javadoc like the above.  It explains nothing and just adds noise.  I usually either add some content (like, what is a Preference used for, what does it mean to set one?) or just delete it if it's obvious.

Reply:
Yeah, I cleaned it up because it was given the name of a node, even though it is not. My guess is there was some copy/paste coding going on here. At least renaming it to give a more clear understanding of what we're working with. Node made no sense here.